data/quarg/quarg news.txt

```diff
@@ -0,0 +1,425 @@
+news "anonymous quarg"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Anonymous Quarg"
+	message
+		word
+			`"Are you what is called a 'tourist'?"`
+	message
+		word
+			`"`
+		word
+			"Are you"
+		word
+			" "
+		word
+			"a merchant"
+			"a merchant captain"
+			"a travelling merchant"
+			"a captain"
+		word
+			"?"
+		word
+			" "
+		word
+			"People"
+		word
+			" "
+		word
+			"of peace"
+			"of law"
+			"of trade"
+		word
+			" "
+		word
+			"are"
+		word
+			" "
+		word
+			"welcome"
+			"appreciated"
+		word
+			" "
+		word
+			"here"
+			"on Quarg worlds"
+			"on all Quarg worlds"
+		word
+			"."
+		word
+			`"`
+news "linguist on quarg planets in human and hai space"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Linguist"
+	message
+		word
+			`"`
+		word
+			"The Quarg language is so different from ours"
+		word
+			" "
+		word
+			"that"
+		word
+			" "
+		word
+			"most people"
+			"ordinary people"
+		word
+			" "
+		word
+			"could never"
+			"wouldn't be able to"
+			"would never"
+		word
+			" "
+		word
+			"learn it"
+			"understand it"
+			"speak it"
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"I've been studying"
+			"I've studied"
+			"I've been learning"
+		word
+			" "
+		word
+			"the Quarg language"
+		word
+			" "
+		word
+			"for a decade"
+			"for years"
+			"for the most part of my career"
+			"all my life"
+		word
+			", "
+		word
+			"but I still can't pronounce their words"
+			"but I've yet to grasp the logic behind it"
+		word
+			"."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"I'm"
+		word
+			" "
+		word
+			"writing"
+			"working on"
+		word
+			" "
+		word
+			"a study"
+			"an analysis"
+		word
+			" "
+		word
+			"of the Quarg language"
+		word
+			", "
+		word
+			"mainly focusing on the "
+		word
+			"pronunciation"
+			"use"
+		word
+			" of "
+		word
+			"nouns"
+			"verbs"
+			"adjectives"
+			"words"
+			"pronouns"
+		word
+			"."
+		word
+			`"`
+news "tourist on quarg planets in human and hai space"
+	location
+		government "Quarg"
+		attributes "human quarg" "hai quarg"
+	name
+		word
+			"Tourist"
+	message
+		word
+			`"`
+		word
+			"These Quarg look so strange I shiver every time I see one."
+			"This world has the cleanest public toilets I have ever seen."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"I absolutely love the look of these buildings. I'm going to decorate my "
+		word
+			"house"
+			"living room"
+			"apartment"
+			"kitchen"
+			"bedroom"
+		word
+			" after them."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Have you seen"
+		word
+			" "
+		word
+			"one of those"
+			"one of their"
+		word
+			" "
+		word
+			"Ringworlds "
```

Comment:
Seems like a strange message when this could very well appear on a ringworld.

Reply:
Would this work with something like `not attributes "ringworld"`, or is the whole thing to be removed?

Reply:
Well not offering on a ringworld would mean this can only offer on Echo.

Reply:
If I'm correct, it would still appear on at least Echo, Humanika (?), Grakhord, Forpelog and Lagrange. These are all moons, planets and stations under Quarg control.

Reply:
`not attributes "station"` is what you're looking for.